src/utils/node-version.js

```diff
@@ -0,0 +1,15 @@
+export default (() => {
```

Comment:
Let's use the module from npm with same functionality: [node-verson](https://www.npmjs.com/package/node-version) for example.